src/algoliasearch.helper.js

```diff
@@ -902,6 +911,11 @@ AlgoliaSearchHelper.prototype.setQueryParameter = function(parameter, value) {
  * @chainable
  */
 AlgoliaSearchHelper.prototype.setState = function(newState) {
+  if (newState.userToken !== undefined && !isValidUserToken(newState.userToken)) {
+    delete newState.userToken;
```

Comment:
we should leave the invalid token IMO

Reply:
I've talked to the search api team and confirmed that we can leave the invalid token for now.